File: MORM.Java/src/MORM/Java/Models/Venda/Regrafiscal.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "REGRAFISCAL")
public class Regrafiscal extends CollectionItem
{
    @CampoAttribute(Campo = "ID_REGRAFISCAL", Tipo = CampoTipo.Key)
    public Integer Id_Regrafiscal;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "DS_REGRAFISCAL", Tipo = CampoTipo.Req)
    public String Ds_Regrafiscal;
    @CampoAttribute(Campo = "IN_CALCIMPOSTO", Tipo = CampoTipo.Req)
    public String In_Calcimposto;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Regrafiscal.cs
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("REGRAFISCAL")]
    public class Regrafiscal : CollectionItem
    {
        [Campo("ID_REGRAFISCAL", CampoTipo.Key)]
        public int Id_Regrafiscal { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("DS_REGRAFISCAL", CampoTipo.Req)]
        public string Ds_Regrafiscal { get; set; }
        [Campo("IN_CALCIMPOSTO", CampoTipo.Req)]
        public string In_Calcimposto { get; set; }
    }
}